code/script.js

```diff
@@ -1,19 +1,62 @@
-// Start here
-
-// Step 1 - Welcome and introduction
-// Your code goes here
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+  "Welcome to Pizzeria Penguin. Ready to order? - Click 'OK' to begin.")
+
+//user name//
+const userName = prompt("Please enter your name:")
+console.log(userName)
+alert (`Hi ${userName}! Get ready to order! `)
+
+//User selection of food//
+const userChoice = parseInt(
+  prompt(`
+Select your choice:
+1 - Pizza
+2 - Pasta
+3 - Salad
+Please enter the number of your choice
+`)
 )
+console.log("user choice", userChoice, typeof userChoice)
+
+let selectedMeal = ""
+
+if (userChoice === 1) {
+  selectedMeal = "Pizza"
+} else if (userChoice === 2) {
+  selectedMeal = "Pasta"
+} else if (userChoice === 3) {
+  selectedMeal = "Salad"
+} else {
+  alert("Invaild choice, please start again")
+}
+
+console.log("selected meal", selectedMeal)
+alert(`You chose ${selectedMeal} ,exellent choice!`)
 
-// Step 2 - Food choice
-// Your code goes here
+//user selection of age//
+let customerType = prompt ("Is the order for a child or an adult?")
```

Comment:
be a little be more descriptive of what is the action of the user in this case, what should it type? Write an explicit instruction to improve the app usability 👍

Reply:
Well noted and changed.